### DISCUSSION TITLE
Please install settings file settings.yml into ${PREFIX}/etc/searx/

### DISCUSSION THREAD

**Discussion by yurivict** [searx/searx#2550] [Installation issue: custom]
Currently it is installed among the python files.

**Comment by return42** [comment]
> Currently it is installed among the python files.

What you see among the python files is the default setup .. you can create your own /etc/searx/settings.yml ... further read:

- https://searx.github.io/searx/admin/settings.html#settings-location
- https://searx.github.io/searx/admin/installation-searx.html#configuration
